interface/patient_file/encounter/find_code_dynamic.php

```diff
@@ -226,76 +226,83 @@ function SelectList(jobj) {
 </head>
 
 <body id="codes_search" class="body_top">
-
-<?php
-$string_target_element = empty($target_element) ? '?' : "?target_element=" . attr_url($target_element) . "&";
-?>
-
-<form method='post' name='theform'>
-<?php
-echo "<p>\n";
-if ($what == 'codes') {
-    if (isset($allowed_codes)) {
-        if (count($allowed_codes) == 1) {
-            echo "<input type='text' name='form_code_type' value='" . attr($codetype) . "' size='5' readonly>\n";
-        } else {
-            echo "<select name='form_code_type' onchange='oTable.fnDraw()'>\n";
-            foreach ($allowed_codes as $code) {
-                echo " <option value='" . attr($code) . "'>" . xlt($code_types[$code]['label']) . "</option>\n";
+    <div class="container-fluid">
+        <?php
+            $string_target_element = empty($target_element) ? '?' : "?target_element=" . attr_url($target_element) . "&";
+        ?>
+
+        <form method='post' name='theform'>
+            <?php
+            echo "<div class='form-group row mb-3'>\n";
+            if ($what == 'codes') {
+                if (isset($allowed_codes)) {
+                    if (count($allowed_codes) == 1) {
+                        echo "<div class='col'><input type='text' name='form_code_type' value='" . attr($codetype) . "' size='5' readonly /></div>\n";
+                    } else {
+                        echo "<div class='col'><select name='form_code_type' onchange='oTable.fnDraw()'>\n";
+                        foreach ($allowed_codes as $code) {
+                            echo " <option value='" . attr($code) . "'>" . xlt($code_types[$code]['label']) . "</option>\n";
+                        }
+                        echo "</select></div>\n";
+                    }
+                } else {
+                    echo "<div class='col'><select class='form-control' name='form_code_type' onchange='oTable.fnDraw()'>\n";
+                    foreach ($code_types as $key => $value) {
+                        echo " <option value='" . attr($key) . "'";
+                        echo ">" . xlt($value['label']) . "</option>\n";
+                    }
+                    echo " <option value='PROD'";
+                    echo ">" . xlt("Product") . "</option>\n";
+                    echo "   </select></div>\n";
+                }
+                echo "&nbsp;&nbsp;\n";
+                echo "<div class='col'>";
+                echo "<input type='checkbox' name='form_include_inactive' value='1' onclick='oTable.fnDraw()' />" .
+                xlt('Include Inactive') . "\n";
+                echo "&nbsp;&nbsp;\n";
```

Comment:
Hello @bradymiller and @tywrenn 

Just asking, such as `&nbsp;` and `\n` character, Are they can remove?

Reply:
Since this is php, do not remove the "\n" character as this is used to escape new lines in php. As far as `&nbsp;` goes, this is used to add some space between characters in html. Since Bootstrap usually takes care of this (and you can also take care of this through padding and just adding a space between text), let's try removing the `&nbsp;` character. I recommend after removing it though to make sure this doesn't change the format of the HTML code and to adjust accordingly.